src/interfaces.js

```diff
@@ -67,3 +68,29 @@ interface DynamicComponentGetViewStrategy {
     */
     getViewStrategy(): string|ViewStrategy;
 }
+
+/**
+* An optional interface describing the configuration object that can be specified to customize bindable properties.
+*/
+interface BindablePropertyConfig {
+  /**
+  * The default binding mode of the property.
+  */
+  defaultBindingMode?: bindingMode;
```

Comment:
I think this will need to be number, since bindingMode is actually an object, not an enum or union type.

Reply:
It's not that simple.
From usage it should be union type.
But I can't see how to make it possible